src/python/dxpy/cli/cp.py

```diff
@@ -151,7 +151,8 @@ def cp(args):
                                     {"objects": src_objects,
                                      "folders": src_folders,
                                      "project": dest_proj,
-                                     "destination": dest_path})['exists']
+                                     "destination": dest_path,
+                                     "targetFileRelocation": str(args.target_file_relocation).lower()})['exists']
```

Comment:
Is the field always safe to provide as input regardless of the file being a symlink?

Reply:
Yes there is logic on the apiserver side to check for it.